### PR TITLE
[Merged by Bors] - chore: move pointwise set files to `Algebra._.Pointwise`

### PR DESCRIPTION
**Pull request by YaelDillies**
---

The remaining files depend on `Data.Set.Pointwise.SMul`, which I intend on reworking.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Order/Group/Pointwise/Bounds.lean [hunk 1]
```diff
@@ -48,6 +48,8 @@ lemma BddAbove.mul (hs : BddAbove s) (ht : BddAbove t) : BddAbove (s * t) :=
 lemma BddBelow.mul (hs : BddBelow s) (ht : BddBelow t) : BddBelow (s * t) :=
   (Nonempty.mul hs ht).mono (subset_lowerBounds_mul s t)
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Order/Group/Pointwise/Bounds.lean:50]: Please split this out

File: Mathlib/Algebra/Group/Pointwise/Finset/Basic.lean [hunk 2]
```diff
@@ -5,11 +5,11 @@ Authors: Floris van Doorn, Yaël Dillies
 -/
 import Mathlib.Algebra.BigOperators.Group.Finset
 import Mathlib.Algebra.Group.Action.Pi
+import Mathlib.Algebra.Group.Pointwise.Set.Finite
 import Mathlib.Data.Finset.Density
 import Mathlib.Data.Finset.Max
 import Mathlib.Data.Finset.NAry
-import Mathlib.Data.Set.Pointwise.Finite
-import Mathlib.Data.Set.Pointwise.ListOfFn
+import Mathlib.Algebra.Group.Pointwise.Set.ListOfFn
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Group/Pointwise/Finset/Basic.lean:12]: Nit: sort imports

File: Mathlib/Algebra/Order/Group/Pointwise/Bounds.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Algebra/Order/Group/Pointwise/Bounds.lean:50]: Okay sure... I've opened #19161, but the current PR is already tiny and will now for sure conflict with #19161

**Comment by kim-em** [comment]
bors merge

**Pull request closed without merging**